charts/zookeeper-operator/templates/operator.yaml

```
@@ -39,10 +46,16 @@ spec:
               fieldPath: metadata.name
         - name: OPERATOR_NAME
           value: {{ template "zookeeper-operator.fullname" . }}
+        {{- if .Values.logForward.create }}
+        {{- include "fluent-bit-sidecar-pod-uid" . | indent 8 }}
+        {{- end }}
         {{- if .Values.resources }}
         resources:
 {{ toYaml .Values.resources | indent 10 }}
         {{- end }}
+{{- if .Values.logForward.create -}}
+{{- include "fluent-bit-sidecar" . | indent 6 }}
```

remove fluent-bit references
```suggestion
{{- include "chart.additionalSidecars" . | indent 6 }}
```

---

Added chart.additionalSidecars and removed fluent-bit-sidecar